cluster-install/install.sh

```diff
@@ -36,7 +36,7 @@ _PROCESSED_LABELS=""
 _PROCESSED_RESOURCE_LIMITS=""
 
 # ------------ CONSTANTS ------------
-readonly __RELEASE_VERSION="24.02"
+readonly __RELEASE_VERSION="${RELEASE_VERSION_OVERRIDE:-24.02}"
```

Comment:
Small bonus change for devs: changing this allows you to downgrade Trident (with some little quirks depending on various factors, but I've done it and it mostly works), or upgrade to a non-default version, such as from 23.07 to 23.10 or something like that.

Reply:
Hmm, you're right, that's basically all it is until (and if) we change the tagging scheme for the connector/neptune stuff. Will make the change.